lib/Conch.pm

```diff
@@ -96,12 +95,12 @@ sub startup {
 
 			if ($code >= 400) {
 				$msg = "$msg || Payload: ".Mojo::JSON::to_json($payload);
-				$c->log->warn($msg);
+				$c->app->log->warn($msg);
```

Comment:
This line has an operational difference.
And now, anytime `$self` appears in the helper, it is the original app, via a closure. (This would be a memory leak if the status sub was ever generated more than once, which I don't think it is in mojo.)

Reply:
That'll get cleaned up in the PR coming from sungo/moar-logging